examples/llama/pippy_llama.py

```diff
@@ -0,0 +1,109 @@
+# Minimum effort to run this example:
+# $ torchrun --nproc-per-node 2 pippy_llama.py
```

Comment:
```suggestion
# $ pip install transformers
# $ pip install accelerate
# $ torchrun --nproc-per-node 2 pippy_llama.py
```

Reply:
accelerate is not needed for this example. It is needed when you want to use it to automatically figure out the split points.

Reply:
Added the pip command.